libs/styles/styles.css

```diff
@@ -1,8 +1,9 @@
 /*** Variables ***/
 
  :root {
-  --global-height-nav: 64px;
+  --global-height-nav: 65px;
```

Comment:
Some consumers might rely on the fact it's 64px and get a tiny amount of CLS https://github.com/search?q=org%3Aadobecom+64px&type=code&p=1

Should we shrink the nav itself by 1px so we are at 64px with the border?

Reply:
I need to look into this a bit more, good callout

Reply:
I think this would grant a bit more time to investigate, since no one should be manually adding `64px` in their projects and instead use `var(--global-height-nav)`. However, there are too many occurrences to enforce this rule, so I'm 'shrinking' the nav by `1px` for now.

This actually addresses a production bug, where the page content overlaps the Global Navigation for `1px.`